plugin/v2/plugin_linux.go

```diff
@@ -6,8 +6,11 @@ import (
 	"runtime"
 	"strings"
 
+	"github.com/containerd/containerd/pkg/userns"
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/oci"
+	rootlessmountopts "github.com/moby/buildkit/util/rootless/mountopts" // TODO: move to moby/sys
+	"github.com/moby/buildkit/util/strutil"
```

Comment:
Slightly concerned about introducing buildkit utils as dependency here; looks like we might be able to either use a local implementation, or our internal sliceutil
https://github.com/moby/moby/blob/77b05b97f43a1dfa5854b9ea64248a8d6c3c3046/internal/sliceutil/sliceutil.go#L6

For the latter, we'll have to add a `go:build` directive though 😞 (for generics), like https://github.com/moby/moby/blob/77b05b97f43a1dfa5854b9ea64248a8d6c3c3046/daemon/container_operations.go#L1-L2

Reply:
fixed